gcs_light_client.mediawiki

```diff
@@ -0,0 +1,992 @@
+<pre>
+BIP: ???
+Layer: Peer Services
+Title: Compact Client Side Filtering for Light Clients
+Author: Olaoluwa Osuntokun <laolu32@gmail.com>
+        Alex Akselrod <alex@akselrod.org>
+Comments: ???
+Comments-URI: ???
+Type: Standards Track
+Created: 05-24-2017
+License: PD
+</pre>
+
+== Abstract ==
+
+This BIP describes a new light client node type for Bitcoin as well as the
+modifications to current full-nodes required to support this new type of light
+client. The light client mode described in this BIP is meant to supersede BIP
+37 as it provides a greater degree of privacy, utility, and also reduces the
+resources required for full-nodes to service this new light client mode
+compared to BIP 37. The light client mode described in this BIP can be seen as
+a "reversal"[1] of BIP 37: rather than the light clients sending filters to
```

Comment:
`[1]` should be `<ref>https://github.com/bitcoin/bips/blob/master/bip-0037.mediawiki</ref>` (and use `<references/>` below) to make it auto-linkable. (suggestion, not mandatory)

Reply:
Done.